examples/usecases/ecommerce-session-based-next-item-prediction-for-fashion.ipynb

```diff
@@ -70,7 +70,7 @@
     "\n",
```

Comment:
<p>The MRR drops from 0.145 to 0.0922 -&gt; this is 37% lower than before.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/849/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>

Reply:
<p>I think MRR before is <code>0.1185</code> not really <code>0.145</code> . Not sure why it printed it out different in the current example? do you know the reason?  </p>

Reply:
In the new nb I only train for 3 epochs, whereas the old one trained for 10 epochs. I am testing with 10 epochs now with the same hyper-params. let's see.